.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: Pull request checks
+on:
+  pull_request:
+    branches: [ master ]
```

Comment:
This will successfully run the CI on each Pull Request.
The linked issue  #3 also meant to run CI when merging to the main branch.
This can be done by adding the following after line 4:
```
 push:
    branches: [main]
```

Reply:
Additionally the main branch is named `main` not `master`.
Could you fix the name please?

Reply:
Fixed